settings.gradle.kts

```diff
@@ -3,6 +3,7 @@ rootProject.name = "zero"
 include(
     "zorm",
     "ztor",
+    "joshi",
```

Comment:
It would be nice to add a README.md to /joshi to explain what it's purpose is

(it's for sharing objects between the data portal frontend and backend which don't need to be shared with AnyLogic)